styleguide/derek/_partials/_footer.jade

```diff
@@ -1,2 +1,2 @@
-.container-fluid.bg-blue-gray.standard-padding
+.container-fluid.bg-blue-gray.fluid-footer-container
```

Comment:
I don't really agree with the way `_navbar` includes the `_ceiling` and `_mainnav` either, but at least `_navbar` and `_mainnav` are distinct names and indicative of what they are.

Having two files named `_footer`, regardless of the different extensions that have virtually no semantic meaning to what they are in the styleguide, is confusing, and confusing is not "easier."

Let's consolidate those files now so that we don't have to do it in the future.